openedx/frontend-app-course-authoring
# feat: add accessibility page

JIRA Ticket: [TNL-10754](https://2u-internal.atlassian.net/browse/TNL-10754)

This PR adds the code for the Accessibility request page. This additional support the [DEPR-381](https://github.com/openedx/studio-frontend/issues/381). 

Testing
1. Set `SHOW_ACCESSIBILITY_PAGE` in env.dev to true
2. Navigate to http://localhost:2001/accessibility
3. Compare the content to http://localhost:18010/accessibility
4. Submit button should be disabled until all fields are valid
5. Fill out the form and check that the email validation is correct and all fields are required
6. Click Submit
7. Should see success alert
